src/App.vue

```diff
@@ -1,141 +1,65 @@
-        ></v-text-field>
-        <v-spacer></v-spacer>
-      </v-app-bar>
-      <v-navigation-drawer
-        v-model="drawer"
-        app
-        clipped
-      >
-        <v-list
-          dense
-        >
-          <template v-for="(item, i) in items">
-            <v-row
-              v-if="item.heading"
-              :key="i"
-              align="center"
-            >
-              <v-col cols="6">
-                <v-subheader v-if="item.heading">
-                  {{ item.heading }}
-                </v-subheader>
-              </v-col>
-              <v-col
-                cols="6"
-                class="text-right"
-              >
-                <v-btn
-                  small
-                  text
-                >edit</v-btn>
-              </v-col>
-            </v-row>
-            <v-divider
-              v-else-if="item.divider"
-              :key="i"
-              dark
-              class="my-4"
-            ></v-divider>
-            <v-list-item
-              v-else
-              :key="i"
-              link
-            >
-              <v-list-item-action>
-                <v-icon>{{ item.icon }}</v-icon>
-              </v-list-item-action>
-              <v-list-item-content>
-                <v-list-item-title class="grey--text">
-                  {{ item.text }}
-                </v-list-item-title>
-              </v-list-item-content>
-            </v-list-item>
-          </template>
-        </v-list>
-      </v-navigation-drawer>
-      <v-content>
-        <v-container
-          fluid
-          class="grey lighten-4 fill-height"
-        >
-          <v-row
-            justify="center"
-            align="center"
-          >
-            <v-col class="shrink">
-              <v-tooltip right>
-                <template v-slot:activator="{ on }">
-                  <v-btn
-                    :href="source"
-                    icon
-                    large
-                    target="_blank"
-                    v-on="on"
-                  >
-                    <v-icon large>mdi-code-tags</v-icon>
-                  </v-btn>
-                </template>
-                <span>Source</span>
-              </v-tooltip>
-            </v-col>
-          </v-row>
-        </v-container>
-      </v-content>
-    </v-app>
+  <!-- id="keep" has no effect ? -->
+  <v-app id="app">
+    <!-- @updateDrawer="handleUpdate" -->
+    <v-content>
+      <top-menu-bar :drawer.sync="drawer"></top-menu-bar>
+      <drawer-menu-bar :drawer.sync="drawer"></drawer-menu-bar>
+      <ShowingLists></ShowingLists>
+    </v-content>
   </v-app>
-</div>
 </template>
-<style>
-#keep .v-navigation-drawer__border {
-display: none
-}
-</style>
+
 <script>
-// import HelloWorld from './components/HelloWorld.vue';
+import TopMenuBarVue from './components/TopMenuBar.vue';
+import DrawerMenuVue from './components/DrawerMenu.vue';
+import ShowingLists from './components/ShowingLists.vue';
```

Comment:
ShowingLists is an action. Lists is the noun. Please refactor this.

Reply:
You can use Lists or ListsView. I prefer Lists, because that's what it shows. The View suffix is implied.